test/FunctionalTestUtils/TelemetryTestsBase.cs

```diff
@@ -80,13 +80,16 @@ public void ValidateBasicException(InProcessServer server, string requestPath, E
             Assert.NotEmpty(actual.Context.Operation.Id);
         }
 
-#if NET451
         public void ValidateBasicDependency(string assemblyName, string requestPath, Func<IWebHostBuilder, IWebHostBuilder> configureHost = null)
         {
             DependencyTelemetry expected = new DependencyTelemetry();
             expected.ResultCode = "200";
             expected.Success = true;
+#if NET451
             expected.Name = requestPath;
+#else
+            expected.Name = "GET " + requestPath;
```

Comment:
Per Sergey's comment on the issue, this is true for event source but in .NET Core aren't we using the diagnostic listener even for full framework?